src/PointwiseFunctions/Hydro/Temperature.hpp

```diff
@@ -0,0 +1,41 @@
+
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "PointwiseFunctions/Hydro/TagsDeclarations.hpp"
+
+namespace hydro {
+
+template <typename DerivedSolution>
+class TemperatureInitialization {
```

Comment:
Could you add a doxygen comment just so the class shows up in the documentation?